lib/classes/Service.js

```diff
@@ -93,6 +93,7 @@ class Service {
         that.plugins = serverlessYml.plugins;
         that.resources = serverlessYml.resources;
         that.functions = serverlessYml.functions;
+        that.runtime = serverlessYml.runtime;
```

Comment:
I'm easy on the outcome. I agree that in reality it's just the stage/region that are the most required. I consider these convenience vars so you don't have to repeat yourself. The fact you can override them means I don't see what harm there is in having it there?